src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -277,90 +320,118 @@ const WorkspaceDashboard = _.flow(
+        initialOpenState: ownersPanelOpen,
+        onClick: () => setOwnersPanelOpen(!ownersPanelOpen)
+      }, [
+        div({ style: { margin: '0.5rem' } },
+          _.map(email => {
+            return div({ key: email, style: { overflow: 'hidden', textOverflow: 'ellipsis', marginBottom: '0.5rem' } }, [
+              h(Link, { href: `mailto:${email}` }, [email])
+            ])
+          }, owners))
       ]),
-      !_.isEmpty(authorizationDomain) && h(Fragment, [
-        div({ style: Style.dashboard.header }, ['Authorization Domain']),
-        div({ style: { marginBottom: '0.5rem' } }, [
+      !_.isEmpty(authorizationDomain) && h(RightBoxSection, {
+        title: 'Authorization domain',
+        initialOpenState: authDomainPanelOpen,
+        onClick: () => setAuthDomainPanelOpen(!authDomainPanelOpen)
+      }, [
+        div({ style: { margin: '0.5rem 0.5rem 1rem 0.5rem' } }, [
           'Collaborators must be a member of all of these ',
           h(Link, {
             href: Nav.getLink('groups'),
             ...Utils.newTabLinkProps
           }, 'groups'),
           ' to access this workspace.'
         ]),
-        ..._.map(({ membersGroupName }) => div({ style: styles.authDomain }, [membersGroupName]), authorizationDomain)
+        ..._.map(({ membersGroupName }) => div({ style: { margin: '0.5rem', fontWeight: 500 } }, [membersGroupName]), authorizationDomain)
       ]),
-      div({ style: { margin: '1.5rem 0 1rem 0', borderBottom: `1px solid ${colors.dark(0.55)}` } }),
-      div({ style: { fontSize: '1rem', fontWeight: 500, marginBottom: '0.5rem' } }, [
-        'Google Bucket'
-      ]),
-      div({ style: { marginBottom: '0.5rem', display: 'flex' } }, [
-        div({ style: { marginRight: '0.5rem', fontWeight: 500 } }, ['Name:']),
-        h(TooltipCell, { style: { marginRight: '0.5rem' } }, [bucketName]),
-        h(ClipboardButton, { text: bucketName, style: { marginLeft: '0.25rem' } })
-      ]),
-      div({ style: { marginBottom: '0.5rem', display: 'flex' } }, [
-        div({ style: { marginRight: '0.5rem', fontWeight: 500 } }, ['Location:']),
-        bucketLocation ? h(Fragment, [
-          div({ style: { marginRight: '0.5rem' } }, [flag]),
-          regionDescription
-        ]) : 'Loading...'
-      ]),
-      h(Link, {
-        ...Utils.newTabLinkProps,
-        href: bucketBrowserUrl(bucketName)
-      }, ['Open in browser', icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })])
+      h(RightBoxSection, {
+        title: 'Tags',
+        info: span({ onClick: e => e.stopPropagation() }, [
+          h(InfoBox, { style: { marginLeft: '0.25rem' } }, [
+            `${getAppName()} is not intended to host personally identifiable information. Do not use any patient identifier including name,
+            social security number, or medical record number.`
+          ]),
+          (busy || !tagsList) && tagsPanelOpen && spinner({ size: '1rem', style: { marginLeft: '0.5rem' } })
```

Comment:
This spinner is slightly taller than the elements beside it, so it causes the select field to shift down when it is visible. This is easiest to reproduce by deleting a tag. Can you make it so there is no shifting?